specification/communication/data-plane/Identity/stable/2022-10-01/CommunicationIdentity.json

```
@@ -314,6 +316,11 @@
           "items": {
             "$ref": "#/definitions/CommunicationIdentityTokenScope"
           }
+        },
+        "expiresInMinutes": {
+          "format": "int32",
+          "description": "Optional custom validity period of the token within [60,1440] minutes range. If not provided, the default value of 1440 minutes (24 hours) will be used.",
+          "type": "integer"
```

You can express the constraints and default formally:
```suggestion
          "type": "integer",
          "minimum": 60,
          "maximum": 1440,
          "default": 1440
```